spec/presenters/section_presenter_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+
+describe SectionPresenter do
+  let(:section) { double(:section, body: '', attributes: {}, attachments: []) }
+  subject { described_class.new(section) }
+
+  [:slug, :title, :summary, :valid?, :errors].each do |method|
+    it "delegates #{method} to the section" do
+      expect(section).to receive(method)
+      subject.send(method)
+    end
+  end
+
+  describe '#body' do
+    context 'a section with a footnotes div' do
+      before do
+        allow(section).to receive(:body).and_return('<div class="footnotes">')
+      end
+
+      it 'adds a title before the existing footnotes' do
+        expect(subject.body).to eq("<h2 id=\"footnotes\">Footnotes</h2><div class=\"footnotes\">\n</div>\n")
```

Comment:
Minor: I'd prefer to use `%{<h2 id="footnotes">Footnotes</h2><div class="footnotes">\n</div>\n}` here to avoid having to escape the double-quotes which I think makes the text less readable.